cli/tools/registry/pm.rs

```diff
@@ -337,9 +339,7 @@ pub async fn add(
   // make a new CliFactory to pick up the updated config file
   let cli_factory = CliFactory::from_flags(flags);
   // cache deps
-  if cli_factory.cli_options()?.enable_future_features() {
-    crate::module_loader::load_top_level_deps(&cli_factory).await?;
-  }
+  crate::module_loader::load_top_level_deps(&cli_factory).await?;
```

Comment:
@bartlomieju do you know why we feature gated this? I feel like we should be running this even without DENO_FUTURE?